README.md

```diff
@@ -318,7 +339,7 @@ install this earlier, you can do so with the command:
 sudo apt-get install node-uglify
 ```
 
-## FRDM-K64F Platform
+## FRDM K64F Platform
```

Comment:
If you look, the link already works, like this:
https://github.com/01org/zephyr.js#frdm-k64f-platform

So I think it would just work w/ the same one you used, with dashes.